src/python/pants/backend/project_info/rules/fast_dependencies_test.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import List, Optional
+
+from pants.backend.project_info.rules import fast_dependencies
+from pants.backend.python.targets.python_library import PythonLibrary
+from pants.build_graph.build_file_aliases import BuildFileAliases
+from pants.testutil.console_rule_test_base import ConsoleRuleTestBase
+
+
+class FastDependenciesTest(ConsoleRuleTestBase):
+  goal_cls = fast_dependencies.Dependencies
+
+  @classmethod
+  def alias_groups(cls) -> BuildFileAliases:
+    return BuildFileAliases(
+      targets={
+        "python_library": PythonLibrary,
+      },
+    )
+
+  @classmethod
+  def rules(cls):
+    return super().rules() + fast_dependencies.rules()
+
+  def create_python_library(
+    self, path: str,
+    *,
+    dependencies: Optional[List[str]] = None
+  ) -> None:
+    self.create_library(
+      path=path,
+      target_type="python_library",
+      name="target",
+      sources=[],
```

Comment:
I think you may be able to leave this off. Check the definition of `create_library` if it has a default arg already.